src/WebJobs.Script.WebHost/Security/KeyManagement/SecretManager.cs

```diff
@@ -608,17 +642,10 @@ private Key CreateKey(string name, string secret)
             return _keyValueConverterFactory.WriteKey(key);
         }
 
-        internal static string GenerateSecret()
+        internal static string GenerateIdentifiableSecret(ulong seed)
         {
-            using (var rng = RandomNumberGenerator.Create())
-            {
-                byte[] data = new byte[40];
-                rng.GetBytes(data);
-                string secret = Convert.ToBase64String(data);
-
-                // Replace pluses as they are problematic as URL values
-                return secret.Replace('+', 'a');
-            }
+            string secret = IdentifiableSecrets.GenerateBase64Key(seed, 40, AzureFunctionsSignature, encodeForUrl: true);
+            return secret.EndsWith("==") ? secret : secret + "==";
```

Comment:
Why the suffix logic "=="?

Reply:
Right, so we ran some tests and the absence of this trailing padding results in a significant number of matches against your core pattern (at GitHub public OSS scale). Adding the trailing padding will go a long way to reduce false positives for this preliminary match. We took a change to Security.Utilities to bring this configurability (padding or no padding) for URL friendly tokens into the API. I will update this PR package version and consume that new API.